packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/ExpertOptions/ExpertOptionsContainer.js

```diff
@@ -1,25 +1,28 @@
-import PropTypes from 'prop-types'
+import { bool, func } from 'prop-types'
+import { useEffect } from 'react'
+
 import ExpertOptions from './ExpertOptions'
 
-function ExpertOptionsContainer ({ storeDemoMode, setDemoMode, ...rest }) {
-  const showDemoModeToggle = window?.location?.search?.includes('demo=true')
+const DEFAULT_HANDLER = () => {}
 
-  if (showDemoModeToggle && storeDemoMode === undefined) {
-    setDemoMode(true)
-  }
```

Comment:
This was the source of the "bad setState call" console error that's been generated by ExpertOptionsContainer forever.